### DISCUSSION TITLE
add templates for new projects.

### DISCUSSION THREAD

**Discussion by RakibRyan** [godotengine/godot-proposals#9154] [Editor]
### Describe the project you are working on

just trying out different ideas 

### Describe the problem or limitation you are having in your project

have to setup the plugins everytime i want to try out a new idea.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

add templates / option that I can set up to automatically activate/install the plugins.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

.

### Is there a reason why this should be core and not an add-on in the asset library?

.

**Comment by AThousandShips** [comment]
Converted to a discussion as there were no concrete ideas for implementation and only limited details

**Comment by KoBeWi** [comment]
You can setup a project and store it somewhere, so making a new one would be a copy-paste of your "template".
